src_py/__pyinstaller/hook-pygame.py

```
@@ -37,9 +37,15 @@ def _append_to_datas(file_path):
 
     pre_binaries = collect_dynamic_libs("pygame")
     binaries = []
+    sdl_major = pygame.get_sdl_version()[0]
```

Flake8 (7.3.0)

[error] undefined name 'pygame'

(F821)